### ISSUE TITLE
Request for the paper

### ISSUE THREAD

**Issue by nibtehaz** [wzy-2020/MMSMAPlus#1]
Hi, your work seems very interesting. Is it possible to share the paper or preprint? Thanks a lot.

**Comment by wzy-2020** [comment]
Of course, you can get our paper in briefings in bioinformatics journal. We have updated the article information in the readme.md file.

**Comment by nibtehaz** [comment]
I actually came from the bib journal page. Unfortunately, my university doesn't subscribe for bib. Would you kindly mail me the paper to nibtehaz@purdue.edu ?
